src/components/List/ApplicationsList.tsx

```diff
@@ -22,6 +24,7 @@ const ApplicationsList: React.FC<ApplicationsListProps> = ({
   loading,
   refetch,
 }) => {
+
```

Comment:
These are just whitespace changes, so can you revert the file to what it was?

Reply:
Sounds good